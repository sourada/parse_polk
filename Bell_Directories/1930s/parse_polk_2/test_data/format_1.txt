4_ R L POLK & CO____
ADAMS JOE A (DOROTHY I) CARP JAMES THOMPSON & SONS H615
10TH ST
ADAMS JOHN F (BARBARA J) BUSINESS MGR COLLEGIATE MFG CO
H816 DUFF AV	<
ADAMS JOHN J (MARIETTA B) (THE YARN SHOP) R MASON CITY
IOWA
ADAMS JOHN R (STELLA L) (AMES BODY SHOP) H4101 DAWES DR
ADAMS KENNETH STUDT R623 6TH ST
ADAMS LYDIA A (WID CHEV V) OFC SEC COLLEGIATE MFG H1518
13TH ST
ADAMS MARIETTA B MRS (THE YARN SHOP) R MASON CITY IOWA
ADAMS MARY C KEY PUNCH OPR IOWA STATE UNIV R RD 2
ADAMS ORVOL A (MARGARET A) E.4P MEMORIAL UNION H623 6TH
ST	*
ADAMS PAMELA A	STUDT R928 BURNETT AV
ADAMS RACHEL I	MRS STEN I S U H441 E 6TH ST
ADAMS ROSEMARY	L OFC SEC HOUSE OF APPLIANCE R412K MAIN
ADAMS VIRGINIA MRS SLSWN OLIVE'S OF AMES R928 BURNETT AV
ADAMS W NEIL (FRANCES) MEMBER MAYORS SCAT COMMITTEE H711
HODGE AV	_————“"I
AMES’ MOST COMPLETE
A	FOOD MARKET
|| |	7 COMPLETE DEPARTMENTS
24th and Grand	Tel. 232-3481
ADAMS WM M ASSOC PROF IOWA STATE UNIV R RD 3
ADAMSON ELDRED N (MARIANNE) OFC MGR DONN ADAMSON INC
H224 N RUSSELL AV
AOAM SON IRENE MRS COOK AMES HIGH SCH R920 BURNETT AV
ADAM SON KARL J (IRENE E) V RRES OONN-ADAMSON INC H1323
CL ARK A V
AOELANTE FRATERNITY MRS ETHEL WARD HSEM OTHE R 304 WELCH
ADELMAN JAMES R (ROBERTA) MANAGEMENT TRAINEE WARDS H405
E 6TH ST APT 6
ADIGUN EBENEZER 0 STUDT R251 N HYLAND AV
ADKINS EDW L (CHRISTY) DRFTSMN GENL FILTER CO H631 9TH
ST
ADOLPHSON MARVIN STUDT RI45 CAMPUS AV
ADRIAN JAMES M STUDT RI43 N SHELDON AV
ADY EMMETT STUDT R3114 WOODLAND ST	CTTAKI
AEGERTER VERDA L (WID HAROLD F) DIR OF CHRISTIAN
FIRST METHODIST CHURCH HI 118 CURTISS AV
AFOLABI NATHL 0 STUDT H251 N HYLAND AV
AGAN MICHL COOK GEORGE'S PIZZA HOUSE R204*
AGARD DAVID W STUDT R618 CARROLL AV	ARROLL AV
AGARO KENNETH W (LUCILLE A» CARRIER P O H618 CARROLL
AGARD KIM STUDT R618 CARROLL AV
AGARD RICHARD J STUDT R618 CARROLL AV
