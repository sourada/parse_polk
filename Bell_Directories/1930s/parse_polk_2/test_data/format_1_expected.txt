615 10TH ST
816 DUFF AV
4101 DAWES DR
623 6TH ST
1518 13TH ST
623 6TH ST
928 BURNETT AV
441 E 6TH ST
928 BURNETT AV
711 HODGE AV
224 N RUSSELL AV
920 BURNETT AV
405 E 6TH ST
251 N HYLAND AV
631 9TH ST
145 CAMPUS AV
143 N SHELDON AV
3114 WOODLAND ST
1 118 CURTISS AV
251 N HYLAND AV
618 CARROLL AV
618 CARROLL AV
618 CARROLL AV
